mottak-arkiv-service/app/database/repositories/arkivuttrekk_repository.py

```diff
@@ -14,6 +14,14 @@ def create(db: Session, arkivuttrekk: Arkivuttrekk) -> Arkivuttrekk_DBO:
     return dbo
 
 
+def update(db: Session, updated_fields: dict, id_: int) -> Arkivuttrekk_DBO:
+    original: Arkivuttrekk_DBO = db.query(Arkivuttrekk_DBO).get(id_)
+    for key, value in updated_fields.items():
+        setattr(original, key, value)
+    db.commit()
+    return original
```

Comment:
Jeg syntes det er bedre å kalle den `arkivuttrekk` i stedet for `original`.
```suggestion
    arkivuttrekk: Arkivuttrekk_DBO = db.query(Arkivuttrekk_DBO).get(id_)
    for key, value in updated_fields.items():
        setattr(arkivuttrekk, key, value)
    db.commit()
    return arkivuttrekk
```

Reply:
Enig, fiksa det